### PR TITLE
[C-841] Put getFeatureEnabled in storeContext

### PR DESCRIPTION
**Pull request by sliptype**
### Description

Moves `getFeatureEnabled` to storeContext in preparation for notification sagas

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/mobile/src/store/storeContext.ts [hunk 1]
```diff
@@ -4,12 +4,16 @@ import type { CommonStoreContext } from 'audius-client/src/common/store'
...
+import {
+  getFeatureEnabled,
+  remoteConfigInstance
+} from 'app/services/remote-config'
 import { walletClient } from 'app/services/wallet-client'
 
 export const storeContext: CommonStoreContext = {
   getLocalStorageItem: async (key) => AsyncStorage.getItem(key),
   setLocalStorageItem: async (key, value) => AsyncStorage.setItem(key, value),
+  getFeatureEnabled,
```
Comment (dylanjeffers) [inline packages/mobile/src/store/storeContext.ts:16]: so can we use remoteConfigInstance to call getFeatureEnabled, or are they separate things?

File: packages/web/src/components/notification/store/sagas.ts [hunk 2]
```diff
@@ -663,10 +665,6 @@ export function* markAllNotificationsViewed() {
   const audiusBackendInstance = yield* getContext('audiusBackendInstance')
   yield* call(waitForBackendSetup)
   yield* call(audiusBackendInstance.markAllNotificationAsViewed)
-  if (NATIVE_MOBILE) {
```
Comment (dylanjeffers) [inline packages/web/src/components/notification/store/sagas.ts:666]: cool this is going away now! is it safe to do in this intermediate phase?

**Review by dylanjeffers — APPROVED** [review]
looks great, isn't there also the `getFeatureEnabled` in the audius-backend-instance? we using the same one there and here? Also just wanted to double check regarding the NATIVE_MOBILE removals!

File: packages/web/src/components/notification/store/sagas.ts (hunk 2 above)
Comment (sliptype) [reply packages/web/src/components/notification/store/sagas.ts:666]: No this shouldn't have been part of this commit, my b

File: packages/mobile/src/store/storeContext.ts (hunk 1 above)
Comment (sliptype) [reply packages/mobile/src/store/storeContext.ts:16]: They are separate things right now. The `getFeatureFlag` helper checks local storage for the feature flag overrides, but I think I can consolidate it

Comment (sliptype) [reply packages/mobile/src/store/storeContext.ts:16]: Actually we do need both, the one in remoteConfig is used by the feature flag modal to show the default state. The helper is used by actual features to check if they're enabled accounting for overrides. I could return the one from audiusBackendInstance out and update the usages to use that one, but it seems like it makes more sense at the top level of the storeContext. wdyt?

Comment (dylanjeffers) [reply packages/mobile/src/store/storeContext.ts:16]: yeah im good with top level, but was just wondering why it seemed like we had duplicate things. maybe when everything is sorted, we take a loot at context and see if we can consolidate? things like getLocalStorageItem vs localStorage, is another example

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sk-c-841-getfeatureenabled-storecontext

**Pull request merged**